readme.html

```diff
@@ -181,5 +177,24 @@ <h2>Permissions</h2>
             </table>
         </p>
     </p>
-</body>
-</html>
+<p>Port Information</p>
+<p>
+    The app uses HTTP/ HTTPS protocol for communicating with the Zscaler server. Below are the default ports used by Splunk SOAR.
```

Comment:
Add Okta server in place of Zscaler server.

Reply:
Fixed